Directory.Build.props

```diff
@@ -0,0 +1,31 @@
+<Project>
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AvaloniaVersion>11.0.1</AvaloniaVersion>
+    <LangVersion>latest</LangVersion>
+    <Nullable>enable</Nullable>
+    <NoWarn>NU5104, NU1507</NoWarn>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+
+    <Authors>Larymar,SKProCH,Appleneko2001</Authors>
+    <Company>Larymar,SKProCH,Appleneko2001</Company>
+    <PackageProjectUrl>https://github.com/AvaloniaCommunity/Material.Avalonia</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/AvaloniaCommunity/Material.Avalonia</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <AccelerateBuildsInVisualStudio>true</AccelerateBuildsInVisualStudio>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)\.editorconfig" Link=".editorconfig" />
```

Comment:
Whats this gives to us?

Reply:
Links .editorconfig file to all project, so that it will be used for code formatting

Reply:
Rider does this automatically. VS doesn't?

Reply:
Didn't test on Rider. It's better to tell directly which editorconfig to use in case someone has many of them in directories tree. And yes, sometimes VS, or Resharper to be fair - ignores it if it is not put explicitly